apps/api/tests/test_session_api.py

```diff
@@ -234,3 +235,15 @@ def test_create_session_new_participant(experiment):
     session = ExperimentSession.objects.get(external_id=response_json["id"])
     assert session.participant.identifier == "jack bean"
     assert response_json == get_session_json(session)
+
+
+@pytest.mark.django_db()
+def test_end_experiment_session_success(client, session):
+    team = Team.objects.create(name="Test Team")
+    session.team = team
+    session.save()
+    url = f"/api/sessions/{session.external_id}/end_experiment_session/"
+    response = client.post(url)
```

Comment:
you need to use an authenticated client:

```suggestion
    user = experiment.team.members.first()
    client = ApiTestClient(user, experiment.team)
    response = client.post(url)
```